### PR TITLE
update release notes right template support

### PR DESCRIPTION
**Pull request by Shellyber**
update release notes now support the right template for incident fields/types and layouts

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-6077

## Description
This PR changes the way update release notes create the template for incident types, incident fields, and layouts.
Before:
```
#### Incident Fields

- **Microsoft Sentinel Incident Number**
```

Now:
```
#### Incident Fields
##### Microsoft Sentinel Incident Number
- %%UPDATE_RN%%
```

### CODE REVIEW

File: .changelog/4304.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+changes:
+- description: update-release-notes command now support the right template for incident fields, incident types, and layouts.
+  type: feature
```
Comment (ShahafBenYakir) [inline .changelog/4304.yml:3]:
```suggestion
- description: Fixed the **update-release-notes** command to support the right template for incident fields, incident types, and layouts.
  type: bug fix (not sure if this is the right syntax)
```

File: demisto_sdk/commands/common/constants.py [hunk 2]
```diff
@@ -410,10 +410,7 @@ class FileType(StrEnum):
 
 RN_CONTENT_ENTITY_WITH_STARS = [
     FileType.CONNECTION,
-    FileType.INCIDENT_TYPE,
     FileType.REPUTATION,
-    FileType.LAYOUT,
-    FileType.INCIDENT_FIELD,
     FileType.INDICATOR_FIELD,
     FileType.TRIGGER,
     FileType.GENERIC_DEFINITION,
```
[the remark this answers is not in the document]
Comment (Shellyber) [reply demisto_sdk/commands/common/constants.py:416]: I'll check.

[the remark this answers is not in the document]
Comment (Shellyber) [reply demisto_sdk/commands/common/constants.py]: Removed it entirely

**Review by ShahafBenYakir — COMMENTED** [review]
Looks good! what about the validations? Can you check if this standard is enforced by our validate for all of these items?

**Comment by Shellyber** [comment]
@ShahafBenYakir I tested everything with the new validate and the old one.
Everything seems valid after the change

File: demisto_sdk/commands/common/tests/release_notes_test.py [hunk 3]
```diff
@@ -785,7 +785,7 @@ def test_validate_headers(mocker, repo):
     ),
     (
         """#### Incident Fields
-                                  - *test**
+                                  - **test**
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/common/tests/release_notes_test.py:788]: So this is now invalid right?

Comment (Shellyber) [reply demisto_sdk/commands/common/tests/release_notes_test.py:788]: Yes, it will fail

**Pull request merged**